src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -117,14 +98,13 @@ const mapStateToProps = state => {
     backendServices.MESSAGING,
   );
 
-  const { attributes, messages, pagination, sort } = folder;
+  const { attributes, messages, pagination } = folder;
 
   return {
     attributes,
     loading: msgState.loading,
     messages,
     recipients: msgState.recipients.data,
-    sort,
```

Comment:
The `sort` prop was only used to pass to `<SortableTable>`'s `currentSort` prop. `<Table>` won't break if `currentSort` is absent, and the table is configured so that it isn't sortable anyway.